airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/python_connectors.py

```diff
@@ -292,6 +292,18 @@ def get_test_steps(context: ConnectorTestContext) -> STEP_TREE:
                 args=lambda results: {"connector_under_test_container": results[CONNECTOR_TEST_STEP_ID.BUILD].output[LOCAL_BUILD_PLATFORM]},
                 depends_on=[CONNECTOR_TEST_STEP_ID.BUILD],
             ),
+            StepToRun(
+                id=CONNECTOR_TEST_STEP_ID.CONNECTOR_VALIDATION_TESTS,
+                step=ValidationTests(context),
+                args=lambda results: {"connector_under_test_container": results[CONNECTOR_TEST_STEP_ID.BUILD].output[LOCAL_BUILD_PLATFORM]},
+                depends_on=[CONNECTOR_TEST_STEP_ID.BUILD],
+            ),
+            StepToRun(
+                id=CONNECTOR_TEST_STEP_ID.CONNECTOR_LIVE_TESTS,
+                step=LiveTests(context),
+                args=lambda results: {"connector_under_test_container": results[CONNECTOR_TEST_STEP_ID.BUILD].output[LOCAL_BUILD_PLATFORM]},
+                depends_on=[CONNECTOR_TEST_STEP_ID.BUILD],
+            ),
```

Comment:
We have a test step granularity decision to make.
My opinion is that we should have two distincts steps: `ValidationTests` and `RegressionTests`.
They are both sub-suites of live tests s but we might orchestrate these test suites differently according to the connector state (we might allow CI to pass if validation fails for the same reason as on master, but fail it if regression test fail etc.).

In others words: lets not make airbyte-ci aware that the live test suite can be used to run both regression and validation.

```suggestion
            StepToRun(
                id=CONNECTOR_TEST_STEP_ID.CONNECTOR_VALIDATION_TESTS,
                step=ValidationTests(context),
                args=lambda results: {"connector_under_test_container": results[CONNECTOR_TEST_STEP_ID.BUILD].output[LOCAL_BUILD_PLATFORM]},
                depends_on=[CONNECTOR_TEST_STEP_ID.BUILD],
            ),
       
```

Reply:
> but the simplest way to reuse the test artifacts 

We could create a  dagger cache volume on the artifact path which would make sure artifacts are shared between steps.

> avoid pytest rerunning the commands

Do you mean the connector commands?
The connector runner uses dagger under the hood, so if the command inputs did not change between the step their output would be cache. (the behavior has to be tested).

But I hear your :) Then  lets make a single `LiveTest` step and control what test suite run in it (validation/regression) via step arguments.

This delays the complexity of decoupling the test suites in separate steps if we ever want to make our CI logic more granular. But I'm fine with it.